wg-ai-integration/charter.md

```diff
@@ -0,0 +1,107 @@
+# WG AI Integration Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter
+README] and uses the Roles and Organization Management outlined in
+[wg-governance].
+
+## Scope
+
+The AI Integration Working Group focuses on enabling seamless integration of
+AI/ML control planes with Kubernetes, as well as providing standardized
+patterns for deploying, managing, and operating AI applications at scale
+on Kubernetes.
+
+The Working Group will provide a forum for a broad engineering community to
+give feedback to the project on challenges encountered when integrating with
+Kubernetes.
+
+This addresses a broad need with many end-users deploying complex AI systems,
+AI/ML platform providers, Kubernetes distributions, and developers of
+distributed AI applications facing these integration challenges. Standardizing
+solutions in this space benefits the entire Kubernetes ecosystem. Adjacent
+ecosystems could link to the outputs of this WG as a trusted vehicle for
+supporting AI integrations with Kubernetes.
+
+### In scope
+
+* Develop a shared community point of view and associated best practices
+enabling AI agent (or multi-agent) systems to integrate with Kubernetes.
+
+* Provide a forum for intersecting code experimentation in AI integration
+space (to be provided externally of Kubernetes via LF/CNCF) and discussion
```

Comment:
Why do we need that separate from k8s? We have kubernetes-sigs, where several other SIGs/WGs are experimenting, don't we?

Reply:
Absolutely open to a repo in kubernetes-sigs for experimentation. Updated.